src/pages/Blog/index.tsx

```diff
@@ -194,7 +198,17 @@ const Blog: React.FC = () => {
                               image={`/blog/images/${elm.slug}.png`}
                             />
                             <CardContent>
-                              <CustomTag blogLabel={elm.tags} />
+                              <div className={classes.wrapperBlock}>
+                                <CustomTag blogLabel={elm.tags} />
+                                <p className={classes.readTime}>
+                                  <img
+                                    src="../Images/svg/time-five.svg"
+                                    alt={t("header.submitAlt")}
```

Comment:
Can we change the alt text here?

Reply:
If the same block of code is used in multiple places, can we add it to a function and return the code from it and call the function wherever required? So that it will easier to edit/change whenever required in the future.

```
const displayTimeRequiredToRead = () => {
          return (
                     <div className={classes.wrapperBlock}>
                                <CustomTag blogLabel={elm.tags} />
                                <p className={classes.readTime}>
                                  <img
                                    src="../Images/svg/time-five.svg"
                                    alt={t("header.submitAlt")}
                                    className={classes.rightSpacing}
                                  />
                                  {`${readingTime(elm.content)} mins read`}
                                </p>
                     </div>
          );
      };
```